packages/phone-number-privacy/src/index.ts

```diff
@@ -1,6 +1,9 @@
+import { BLINDBLS } from 'bls12377js-blind'
 import * as functions from 'firebase-functions'
+import { PHONE_NUMBER_PRIVACY_SECRET_KEY } from './config'
 
 export const getSalt = functions.https.onRequest((request, response) => {
-  console.log(request.body)
-  response.send(request.body)
+  const privateKey = new Buffer(PHONE_NUMBER_PRIVACY_SECRET_KEY)
+  const salt = BLINDBLS.computePRF(privateKey, new Buffer(request.body.blindPhoneNumber))
+  response.json({ success: true, salt })
```

Comment:
Let's wrap this in a try catch with some more logging. Np if you'd rather do that in a follow-up PR

Reply:
will do